account_permanent_lock_move/models/res_config.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Camptocamp SA (Matthieu Dietrich)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import api, fields, models
+
+
+class AccountConfigSettings(models.TransientModel):
+    _inherit = 'account.config.settings'
+
+    permanent_lock_date = fields.Date(
+        string="Permanent Lock Date",
+        related='company_id.permanent_lock_date',
+        help="Non-revertible closing of accounts prior to and inclusive of "
+        "this date. Use it for fiscal year locking instead of ""Lock Date"".")
```

Comment:
`""` will have no effect (Python will considerate it as 2 strings to concatenate). You should use `\"` or surround the whole string with `'`.